server.js

```diff
@@ -44,4 +53,15 @@ db.once("open", function () {
   console.log("Mongoose is connected");
 });
 
+async function handlePostBooks(req, res) {
+  // console.log(req.body);
+  try {
+    const bookMaker = await Book.create(req.body);
+    res.status(204).send(bookMaker);
```

Comment:
- on line 60, 200 is more commonly used but I think a 204 might work here as well 👍